controllers/platform/properties.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2022 The VolSync authors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+package platform
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	ocpsecurityv1 "github.com/openshift/api/security/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	apimeta "k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Properties contains properties about the environment in which we are running
+type Properties struct {
+	IsOpenShift        bool // True if we are running on OpenShift
+	HasSCCRestrictedV2 bool // True if the StorageContextConstraints "restricted-v2" exists
```

Comment:
typo in comment - StorageContextConstraints should be SecurityContextConstraints?